scripts/linters/other_files_linter.py

```diff
@@ -327,6 +328,48 @@ def _check_that_workflow_steps_have_name(
             for job in jobs_with_unnamed_step
         ]
 
+    def check_compressible_images(self) -> concurrent_task_utils.TaskResult:
+        """Checks if any images in the assets 
+        directory can be compressed further.
+
+        Returns:
+            TaskResult. A TaskResult object 
+            representing the result of the lint check.
+        """
+
+        name = 'Compressible images'
+        repo_path = os.path.join(os.getcwd(), 'assets')
+
+        compressed_images = compress_images.get_compressible_images(repo_path)
+
+        failed = bool(compressed_images)
+        error_messages = []
+
+        if compressed_images:
+            error_messages.append(
+                f'Found {len(compressed_images)} images that could be '
+                'compressed further:')
+            for image in compressed_images:
+                saved = image['original_size'] - image['new_size']
+                saved_percentage = (
+                    (image['original_size'] - image['new_size']) /
+                    image['original_size']) * 100
+                error_messages.append(
+                    f'{image["path"]} - Could save {saved} bytes '
+                    f'({saved_percentage:.2f}%)')
+
+            error_messages.append(
+                '\nUse the following commands to compress the images:')
+            error_messages.append(
+                'For PNG images: gm convert <input_file> -strip '
+                '-compress Zip <output_file>')
+            error_messages.append(
+                'For JPG and WebP images: gm convert <input_file> -strip '
+                '-compress LZW <output_file>')
+
+        return concurrent_task_utils.TaskResult(
+            name, failed, error_messages, error_messages)
```

Comment:
```suggestion
        return concurrent_task_utils.TaskResult(
            'Compressible images', bool(compressed_images), error_messages, error_messages)
```

simplify by avoiding unnecessary intermediate variables